doc/cookbook/source/examples/evaluation/cross_validation.rst

```diff
@@ -0,0 +1,102 @@
+============================================
+Cross Validation
+============================================
+
+Cross validation is a type of model validation technique to estimate an algorithm's performance on unseen data.
+
+It essentially involves training the model on a subset of the data, and testing on the
+remaining data.
+
+The process is repeated for several complementary partitions and
+combines all the validation results generated (For instance, an average).
```

Comment:
grammar error make it
"and all validation results are combined, e.g. by averaging"